src/oscar/core/application.py

```diff
@@ -12,7 +12,7 @@ class Application(object):
     This is subclassed by each app to provide a customisable container for an
     app's views and permissions.
     """
-    #: Namespace name
+    #: Application namespace name
```

Comment:
I think you mean this comment to be "Application name" rather than "Application namespace name", as the latter confuses things even more ;-).